clients/src/main/java/org/apache/kafka/common/security/authenticator/SaslServerAuthenticator.java

```diff
@@ -73,6 +73,7 @@
 
 public class SaslServerAuthenticator implements Authenticator {
 
+    private static final int MAX_RECEIVE_SIZE = 131072; // Chosen as twice the max receive size for GSSAPI
```

Comment:
Hmm. I considered it, but the benefit seemed doubtful. If we exceed the limit on the client, I guess we would throw an exception? I thought it might be better to let the server make that determination in case we find that we need to increase it in the future.